portfolio.html

```diff
@@ -0,0 +1,31 @@
+ <!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="normalize.css" type="text/css">
+    <link rel="stylesheet" href="portfoliostyles.css" type="text/css">
+    <meta name="viewport" content="width=device-width">
+</head>
+<body>
+	<nav class="NavBar">
+		<a class="NavBarLinks" href="#Portfolio">Portfolio</a>
+		<a class="NavBarLinks" href="#AboutMe">About Me</a>
+		<a class="NavBarLinks" href="#Contact">Contact</a>
+	</nav>
+	<section class="IntroSection">
+		<div class="IntroImage">
+			<img src="tetris.jpg" alt="Man Coding" width="450px" height="450">
```

Comment:
why did you specify the width and height in the HTML? could you have done it in the CSS?